src/main/java/mate/controller/car/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,29 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars/current")
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long driverId = (Long) req.getSession().getAttribute(DRIVER_ID);
+        List<Car> allByDriver = carService.getAllByDriver(driverId);
+        req.setAttribute("cars", allByDriver);
+        req.getRequestDispatcher("/WEB-INF/views/cars/GetMyCurrentCars.jsp").forward(req, resp);
```

Comment:
we don't need to create new jsp for displaying current cars, we can use the old one, which displays list of all cars

Reply:
done